### PR TITLE
feat(Bonus Pagamenti Digitali): [#176191591] Open ranking details when tap on ranking preview

### PR DESCRIPTION
**Pull request by fabriziofff**
# This pr depends on #2643 

## Short description
This pr allows to see the ranking bottomsheet details when tap on the ranking item preview.

https://user-images.githubusercontent.com/26501317/103239079-d122e800-494c-11eb-8baf-4ed163b44eed.mov

## List of changes proposed in this pull request
- Add tap on `SuperCashbackRankingReady` in order to open the bottomsheet
- Fixed the number representation for `citizens` and `amount` in `SuperCashbackBottomSheet`

## How to test
Tap on the preview item to open the bottomsheet

### CODE REVIEW

File: locales/it/index.yml [hunk 1]
```diff
@@ -2064,7 +2070,7 @@ bonus:
         rankTitle: "Classifica Super Cashback"
         howItWorks:
           title: "Come funziona?"
-          body: "I **{{citizens}} cittadini** con il più alto numero di transazioni **ricevono il Super Cashback, un premio pari a {{amount}}.**\n\n**La classifica cambia ogni giorno.** Quella definitiva sarà visualizzata il {{endDate}}."
+          body: "I **{{citizens}} cittadini** con il più alto numero di transazioni **ricevono il Super Cashback, un premio pari a {{amount}} €.**\n\n**La classifica cambia ogni giorno.** Quella definitiva sarà visualizzata il {{endDate}}."
```
Comment (thisisjp) [inline locales/it/index.yml:2073]:
```suggestion
          body: "I **{{citizens}} cittadini** con il più alto numero di transazioni **ricevono il Super Cashback, un premio pari a €{{amount}}.**\n\n**La classifica cambia ogni giorno.** Quella definitiva sarà visualizzata il {{endDate}}."
```

File: locales/en/index.yml [hunk 2]
```diff
@@ -2033,7 +2040,7 @@ bonus:
         rankTitle: "Super Cashback ranking"
         howItWorks:
           title: "How does it work?"
-          body: "The **{{citizens}} citizens** with the highest number of transactions **receive the Super Cashback, a prize of {{amount}}.**\n\n**The ranking changes every day.** The final one will be displayed on {{endDate}}."
+          body: "The **{{citizens}} citizens** with the highest number of transactions **receive the Super Cashback, a prize of {{amount}} €.**\n\n**The ranking changes every day.** The final one will be displayed on {{endDate}}."
```
Comment (thisisjp) [inline locales/en/index.yml:2043]:
```suggestion
          body: "The **{{citizens}} citizens** with the highest number of transactions **receive the Super Cashback, a prize of €{{amount}}.**\n\n**The ranking changes every day.** The final one will be displayed on {{endDate}}."
```

File: locales/en/index.yml [hunk 3]
```diff
@@ -1939,12 +1939,19 @@ bonus:
           gracePeriodBody: "This cashback phase ended on {{date}}. We are still waiting for the latest transactions from the banks. Just bear with us: we'll keep you posted here!"
           closedPeriodOKBody: "Congratulations {{name}}! \nYou are entitled to € {{amount}} of Cashback"
           closedPeriodMaxAmount: "it's the highest refund possible!"
+          moneyTransfer: "By {{date}} you will receive the transfer on the IBAN you communicated."
```
Comment (thisisjp) [inline locales/en/index.yml:1942]:
```suggestion
          moneyTransfer: "By {{date}} you will receive the transfer on the IBAN you entered."
```

**Review by thisisjp — CHANGES_REQUESTED** [review]
small proofreading :)

**Review by Undermaken — APPROVED** [review]

**Review by Undermaken — APPROVED** [review]

**Review by Undermaken — APPROVED** [review]

**Pull request merged**